libs/cardano-ledger-binary/cardano-ledger-binary.cabal

```diff
@@ -67,7 +67,7 @@ library
         network,
         nothunks,
         primitive,
-        plutus-ledger-api >=1.27.0 && <1.33.0,
+        plutus-ledger-api >=1.27.0 && <1.34.0,
```

Comment:
Can you be more explicit, please? What exactly should I change here? The `cardano-ledger-binary` version at the top or something else?

Reply:
> The cardano-ledger-binary version at the top or something else?

Yes this package, that the cabal file is for `cardano-ledger-binary` should have `1.3.4.1` as  its version.

Reply:
Should I do that for every other cabal file where I've bumped the plutus versions?

Reply:
I bumped the versions in every cabal file I modified, hope that is what you meant. Please take another look when you have time!

Reply:
No that's not what I meant. I meant only `cardano-ledger-binary` because all other packages that you've modified had already received some sort of changes since they have been released last.